### ISSUE TITLE
Light Theme: Add Blog option

### ISSUE THREAD

**Issue by meetinghouse** [meetinghouse/cms#167]
We would like to make the blog an option in the Light Theme. It should only be accessible to toggle it on or off at Admin level access, and perhaps a checkbox could be placed in the Admin Settings page for that, similar to the "Check to use the dark theme" option available to admins now.

Is this feasible?

**Comment by acavanagh** [comment]
definitely feasible.  Probably take a couple-ish hours to implement.

**Comment by meetinghouse** [comment]
OK, let's do it. Here are some specs, but if you need more specific specs (heh), let me know.

_Please create a radio button toggle in the Admin Settings page, available to admin users only, to turn on or off the blog for the Light Theme. It should default to off.
_When the blog is turned on, please make a link appear in both the left nav (at the bottom) and in the top menu (at the right side). This link should say "Blog,", not "Builder's Notebook."
_The blog needs to match the light theme visual layout, not suddenly jump in appearance to the dark theme layout. Let us know if this isn't something you included in your ballpark quote. 

Thanks.

**Comment by meetinghouse** [comment]
Hello, Andy. The client who requested this item has come back and is still interested. Can you provide an updated estimate based on the specs above, and a timeframe for when this could be done? Thanks.

**Comment by acavanagh** [comment]
Hi John - sorry for the delay on getting back to you on this, I had some
back problems that took me out of commission for a week or so and I'm still
digging out.

I take it from this email that Barry isn't able to be taking over?  I
should be able to make some time for this next weekend.  Estimating 3-4
hours, depending on how much revision we need on the layout for the light
theme.  Also, once this code is merged in this option will be global across
all the sites, so it won't be available only to this client.  We can make
the switch admin only so only you guys will be able to see it though.

**Comment by meetinghouse** [comment]
Andy,

Thanks for your prompt reply, anyway! I hate back problems and hope that
your recover promptly. My extensive experience is that acupuncture works
like nothing else. I have a standup desk, and it helps.

My issue is that we have clients who have asked us for upgrades, backlogged
some months and my answer for them when they ask, "when is my upgrade going
to be completed?" is, "I don't know." That is not a criticism of anybody
but me, it is my failure and my problem.

That said, Barry has been out of commission, working for the USPO holiday
crew, but I met with him recently and he is back in the world. His capacity
is growing, as yet. He is a team with John Bozaan, our designer, and I have
high hopes for their ability. For the present, they will need your
guidance, I am sure, and there will be times when it may be better for you
to just execute a given work order, if you have time to do so. This current
order may be one of those times.

I'll do some work on our end, to see if I can get you some useful guidance.

Best Wishes for the New Year 2016!




Cordially,

John Corbett, President
Corbett Research Group
http://www.restorationtrades.com
"Web Marketing that Works"

Publishers of:

The Custom Building & Restoration Trades Directory
http://www.restorationtradesdirectory.com

The Building Arts Notebook, "Journal of the Guild of Building Artisans"
http://buildingartisansguild.com/

Email:  John@RestorationTrades.com
Phone: 413-586-4748

**Comment by meetinghouse** [comment]
Barry, as you can see, Andy could work on this. Can you also look at it just enough to give a time estimate for yourself? I'm not sure if you or Andy would be best to work on this, but some input from you on time would be helpful in making that decision.

**Comment by barryaprogrammer** [comment]
Hi John,



Useful meeting.



Here is what I see as the next steps.

1)    John and I need to start using Github ourselves. Katherine said that
would be helpful to her. So I will proceed to push in the changes to the
‘basecontroller.php’ using github (for the slideshow).  Review with John so
we are all in synch.

2)    Get a better handle on bootstrap it;'s capabilities as we are using
it. It’s tied to Github on our system.  John started this review as per
email.

3)    Katherine to set up light theme test site and provide username and
password.

4)    Me to look into virtual machine option for testing. (lower priority -
Dan's feedback might be helpful )

5)    All these things tie into #167, #193. Hopefully we’ll have a chance
to touch bases with Andy in the meantime.



I expect #1 done by this weekend. #2 as per John’s earlier email we have
started exploring and it will help lay foundation for the tasks 167, 193.
I’ll follow up on Monday.



Thanks

barry

**Comment by jboz62** [comment]
Closing issue #167, already completed in issue #267.
